Composer/packages/client/src/pages/design/styles.ts

```diff
@@ -73,6 +73,9 @@ export const visualPanel = css`
   flex: 1;
   border-right: 1px solid #c4c4c4;
   position: relative;
+  &:focus {
```

Comment:
Focus is required to have styling so that a keyboard user can always see where they are on the page.